docs/modelling/index.rst

```diff
@@ -0,0 +1,8 @@
+Modelling
```

Comment:
:us: en-US, 1033, enu. Embrace the arbitrary spelling changes made by Noah Webster!